### ISSUE TITLE
spanish translation not work at initial data exploration

### ISSUE THREAD

**Issue by johnfelipe** [metabase/metabase#23218]
![SNAG-1213](https://user-images.githubusercontent.com/428820/172501915-193b13c4-f2b0-4c5b-aa3b-3e87e6834269.png)

![SNAG-1214](https://user-images.githubusercontent.com/428820/172502020-f71d5954-53e8-4fa4-87f6-6fbceb43f5ee.png)

**Comment by johnfelipe** [comment]
latest build today, and remain issues with translate:

![SNAG-0000](https://user-images.githubusercontent.com/428820/177356199-02db77e2-400f-4f97-9f7c-09eabc33c7de.png)

![SNAG-0001](https://user-images.githubusercontent.com/428820/177356384-2a28b865-34bb-443a-822e-315125728a4f.png)

![SNAG-0002](https://user-images.githubusercontent.com/428820/177356469-15f901eb-74b2-46dd-adf1-c8ccd0594051.png)

![SNAG-0003](https://user-images.githubusercontent.com/428820/177356624-57f4d136-93ca-4e4c-9262-63c005e775d6.png)

![SNAG-0004](https://user-images.githubusercontent.com/428820/177356789-5509e3ae-38c7-48ef-bd5b-8c64672e3736.png)

![SNAG-0005](https://user-images.githubusercontent.com/428820/177356891-8819f181-5cfc-4e77-8ac9-c1f28a6b4914.png)

**Comment by iethree** [comment]
I'm pretty sure these strings are generated on the backend

**Comment by darksciencebase** [comment]
yes, they are. [thread](https://metaboat.slack.com/archives/C0641E4PB9B/p1702297934012849)
